### DISCUSSION TITLE
blog/posts/hello-zotero-7

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#261] [文档评论区]
# blog/posts/hello-zotero-7

Zotero 非官方中文维护小组

https://zotero-chinese.com/blog/posts/hello-zotero-7

**Comment by lgyz** [comment]
当我试图安装Zotero-7.0.1-beta.2+ca83e4303_x64_setup的时候，出现如下提示：
“Sorry, Zotero can't be installed. This version of Zotero requires 64-bit
Microsoft Windows 7or newer.”  我的系统是Windows 11，64 位操作系统, 基于 x64 的处理器。
这是什么原因？

**Reply by northword** [reply]
是 Zotero 的问题，请等待更新。另外建议普通用户从官网首页下载正式版安装，而不是使用 beta 版本。

**Reply by lgyz** [reply]
好的，谢谢及时回复。安装最新beta版，问题已经解决。👌

**Comment by 15388513519** [comment]
已经读了的文献，仪表盘一直是0时长0页，期刊标签得点一下，右下角才会显示，然后影响因子必须得一个一个从greenfrog更新才行，这些问题怎么解决呀

**Reply by northword** [reply]
仪表盘问题请向 Chartero 插件反馈。期刊标签请向 Style 插件反馈，Green Frog 插件请确认在设置中已启用“自动更新期刊信息”。

**Comment by Mctalos** [comment]
有没有开发大大能恢复一个zotfile的功能 “Get table of contents” 。这样就能提取书籍、文件的目录大纲。检索文件很方便。
相比搜索“everything”，结果太多了，难以定位要找的文件。

**Comment by tlt18** [comment]
我升级zotero7后，之前用zotero6记的笔记（包括图片和引用文字）点击“在页面显示”无法跳转到pdf对应的位置，而用zotero7记的笔记可以正常跳转。请问这是不是普遍问题？

**Reply by northword** [reply]
我没有遇到类似的情况，也许你可以把这个情况发到官方论坛看看有没有什么解决方案。

**Comment by janicet999** [comment]
zotero7后续会更新模糊搜索功能吗，感觉解析pdf生成条目还不是特别强大

**Reply by northword** [reply]
你是指通过 pdf 获取条目？还是指用关键词模糊搜索文库？

**Comment by Monica99999** [comment]
就是当阅读文献后，标签中有一个阅读时间显示的标记，比如显示你看了几分钟？但是这个阅读时间有时候会自动消失，并且之前的记录都没了。只有你重新看文献，才会重新记录。



发自我的iPhone


------------------ Original ------------------
From: Northword ***@***.***&gt;
Date: Sun,Sep 8,2024 9:39 PM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: Monica99999 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] blog/posts/hello-zotero-7 (Discussion #261)





 
你是指通过 pdf 获取条目？还是指用关键词模糊搜索文库？

**Reply by northword** [reply]
这是插件提供的功能，style 或 chartero，请确保插件处于最新版本，仍有问题可向插件开发者反馈。

**Comment by janicet999** [comment]
是的，我希望我上传pdf后可以根据题目，作者或者关键词找到这篇文章的其他版本，比如我下载的是arxiv版，上传后有个模糊搜索功能可以搜索到这篇文章的cvpr版（类似这个意思）并更新条目信息，不需要下载cvpr版。

发自我的iPhone


------------------ Original ------------------
From: Northword ***@***.***&gt;
Date: Sun,Sep 8,2024 9:39 PM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: Janiceter ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] blog/posts/hello-zotero-7 (Discussion #261)





 
你是指通过 pdf 获取条目？还是指用关键词模糊搜索文库？

**Reply by northword** [reply]
预印本的问题可以参考插件 zotero arxiv workflow 是否能缓解。

**Comment by ttsstt-j** [comment]
zotero7，Windows应用列表里显示，占用空间50g，怎么回事呢？

**Comment by yuelinxt** [comment]
升级到了zotero7，然后导入了epub书籍。发现稍微大一点的epub就无法打开，我试了很久，只打开了5M左右的电子书，非常影响使用啊，请问是否有方案解决？

**Comment by ZQ-starry** [comment]
7版本有没有解决单处插入文献的格式问题？以前只能[1],[2],[3],而不能[1-3]。这个问题困扰好久了，没有好的解决办法。

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/ms-word-plugin#%E5%9C%A8%E5%90%8C%E4%B8%80%E4%BD%8D%E7%BD%AE%E5%BC%95%E7%94%A8%E5%A4%9A%E4%B8%AA%E6%96%87%E7%8C%AE

**Reply by ZQ-starry** [reply]
那有没有可以解决之前插入文献无法实现超链接的解决方案呢？

**Comment by GZMLA** [comment]
请问更新到zotero 7之后，导出文献库的时候卡在那里始终没有响应，可能是哪里出了问题？谢谢

**Reply by northword** [reply]
如果可以稳定复现，请向  https://forums.zotero.org/ 反馈。

**Comment by widcardw** [comment]
刚刚在 mac 上换上了 Zotero 7，且用的是 OneDrive 同步，重新打开 Zotero 7 出现找不到 sqlite 数据库的问题，实际上只要右键 Zotero 在 OneDrive 的文件夹，“始终保持在该设备上”，等待下载完成之后就能正常启动了。

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/sync

**Comment by kkkk45-yyy** [comment]
我从知网中获取文献时只能获取到网页，不能获取到文章和pdf，应该怎么解决

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/faqs/update-translators

**Comment by sheepyounger** [comment]
请问  Zotero 是否有 HarmonyOS 版本的开发计划？出于需要我有换鸿蒙设备的可能，但是文献都存在  Zotero，这对我而言是个麻烦。

**Reply by northword** [reply]
询问官方团队：https://forums.zotero.org/

**Comment by wangyuanwill** [comment]
有没有办法（插件）把zotero阅读时框选的注释全部导出为图片？为了导入anki等软件里面复习！注释预览里面可以一张一张的导出，但是框选的注释多了一张一张的导出太麻烦了，有没有批量导出的？框选的注释zotero存储的文件夹里面是以Png图片保存起来的，但是每个注释都是分散保存，没在一个文件夹里面。

**Comment by littlefish3018** [comment]
Zotero 7注释和复制的英文单词是乱码，请问如何解决？谢谢

**Reply by northword** [reply]
大概率 PDF 的问题，无解

**Reply by littlefish3018** [reply]
谢谢

**Comment by littlefish3018** [comment]
求助！今天所有注释都消失了，也无法新添加注释。

**Reply by littlefish3018** [reply]
网页上可以看到注释痕迹，但软件里看不到。
请帮忙指导，谢谢！

**Reply by wakewon** [reply]
没听说有类似的情况，禁用所有插件，重启后保持插件禁用再试。另外请确保你是在 Zotero 内置阅读器打开的，而不是其他软件。

**Comment by yugang1982** [comment]
求助！！！
类似：https://www.djyanbao.com/report/detail?id=3495534&from=search_list&aiStatus=undefined。
这样的网页的剪照，不会显示出来。
只会显示出很少一部分的内容。

**Comment by dreamkwok** [comment]
为啥我在Zotero看文献时，打开到第6个文献后，会出现PDF加载不出来，就是没东西显示的情况？

**Reply by wakewon** [reply]
没遇到这个规律。禁用所有插件，重启后保持所有插件关闭再试。

**Comment by Lnn1ning** [comment]
能不能把scholaread的插件也收录进去啊，跪求。

**Reply by northword** [reply]
这不是一个开源插件，也没有一个仓库用来发布 XPI 文件，因此我们无法收录此插件。

此外，该插件拥有自己的发布途径，无需商店的镜像即可下载。

**Comment by yugang1982** [comment]
是PDF非加密文献已经下载到本地了吗？



---Original---
From: ***@***.***&gt;
Date: Mon, Dec 2, 2024 13:28 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [zotero-chinese/wiki] blog/posts/hello-zotero-7 (Discussion #261)




 
为啥我在Zotero看文献时，打开到第6个文献后，会出现PDF加载不出来，就是没东西显示的情况？

**Reply by wakewon** [reply]
没遇到这个规律。禁用所有插件，重启后保持所有插件关闭再试。

**Comment by yijian-1118** [comment]
阅读器-链接预览 有关闭的办法嘛，鼠标容易扫到，影响阅读

**Reply by wakewon** [reply]
禁用 Reference 插件

**Comment by xujiakun-bit** [comment]
现在使用zotero7总是卡顿，之前以为是内存问题，由8G加内存条变为16G，内存占用在50%以下，仍然会卡。请问是zotero文献多的原因吗，3000+篇，还是需要更大运行内存？

**Reply by wakewon** [reply]
禁用所有插件再试，通常是插件的问题。确保 Zotero 和插件都更新到了最新版。

**Comment by likerrrrrr** [comment]
各位好，我想问一下，我更新了zotero之后，在word的Add/Edit Citation中我能准确的输入我的文献名称，但是她却在打开文件的最下面，我需要划到最下方才能找到我的文库中的文章，有的时候这个文献我已经下载到zotero中了，划到最下方也没有，需要关掉一次到两次才能搜索到但也需要划到最下面，我想请问两个问题：可不可以精确搜索，把我打字搜索到的文章置顶而不是放在最下面上面是一群无关的文章。第二个是之前版本都可以时时更新这个可以吗，感谢

**Reply by wakewon** [reply]
1. 把“打开的文献”显示在前面是 Zotero 的功能，别在 Zotero 中同时打开过多的文献，不方便实用也影响性能。
2. 我没发现搜不到的问题。
3. 这两个问题如果希望优化，可以去 Zotero 官方论坛发帖提建议，中文社区这边没有 Zotero 官方的开发者，处理不了 Zotero 本身功能的设计问题。

**Comment by Huangleyi-jpg** [comment]
我更新后发现wps和word显示zotero图标，但是点击不跳转，用不了

**Reply by wakewon** [reply]
1. 重装 Word 加载项。详见：[Word 加载项相关教程](https://zotero-chinese.com/user-guide/faqs/word-addon#word-加载项错误相关)
2. 禁用 Citation插件。
3. 重装 Microsoft Word 和最新版 WPS。

**Comment by ninismart** [comment]
打开PDF文件之后，文件名右侧的“开合内容窗格”图标是上下结构，打开之后，条目笔记只能显示在PDF下面，不知道怎么变成这样的，卸载重装还是这样的。怎么变回PDF和条目笔记横向排列？谢谢

**Reply by ninismart** [reply]
解决了，是显示比例设置成了200%，条目笔记被挤出了右边只能在下面。是我太蠢了TT

**Comment by CR-20** [comment]
你好我想请问一下我的zotero在设置默认参考文献格式的时候没有确认键，更新了版本也没有，而且我的zotero中也没有首选项这个功能

**Comment by FiRepRooFC** [comment]
先是在zotero自动更新为Zotero 7,在Zotero软件首页的建议下，从官网下载了Zotero-7.0.24_x64_setup，在安装的过程使用自定义安装，并选择了和之前一样的安装路径。但在重启之后，Zotero 7安装失败，原来的版本也被删除，该文件夹下的内容全部消失了，只剩下之前安装的两个插件。请问这是什么原因呢？

**Reply by FiRepRooFC** [reply]
哎呀，再重新安装一次就可以恢复数据了

**Comment by siri502C** [comment]
我在尝试用word宏代码建立交叉引用链接失败之后，在word中用zotero引用文献时会跳出
“XML Parsing Error: no root element found
Location: chrome://zotero/content/integration/progressBar.xh
Line Number 1, Column 1:”的信息，一闪而过，但是文献引用情况正常，请问这个有解决方法嘛？

**Comment by Winfred2** [comment]
为什么我的安装应用里有两个zotero，占用的内存很大，我可以删掉一个么

**Reply by wakewon** [reply]
> 占用的内存很大

这个是因为你的安装位置选的极为不合适，安装的时候自定义了路径，但没有给Zotero程序单独建立一个文件夹存放文件，导致Zotero程序文件所在目录的所有文件的大小都被统计在内了（比如装到了Program Files根目录，就会导致这个目录下所有软件都被认为是Zotero的一部分，又比如把数据文件夹放到了程序文件夹里，导致文献数据和PDF也被认为是程序文件的一部分）。这种情况最好卸载重装，为程序单独使用一个独立的目录存放。

> 为什么我的安装应用里有两个zotero

这个可能是Zotero本身升级的小问题，但如果你没有自定义安装路径的话是不会遇到的。简单来说就是升级的时候没能覆盖安装，而是另装了一个新的。综合前面占用的问题，建议不要自定义程序目录，毕竟你选的目录很可能非常不合适，不好说未来会不会遇到些抽象问题。

> 我可以删掉一个么

可以，有必要的话两个都卸了，重装个干净的在默认的路径下，别改。

**Comment by Winfred2** [comment]
谢谢你的详细回复。
目前我因为本身C盘空间剩的不是很多，所以当时安装的时候在D盘里专用建了一个zotero的文件夹，下载软件时就把自定义安装在了D盘的这个文件夹里，这样也不可以么？如果我使用默认安装位置的话，我觉得我的C盘会撑不住……只能这一种方法改变占用内存大这一点么？有没有什么办法能先清理一下zotero的缓存？
关于两个zotero能不能卸载，因为我没卸载过，我担心卸载重装后我的文献还会存在么？
谢谢你的回复。
祝好。
---- 回复的原邮件 ----
发件人 ***@***.***> 日期 2026年03月17日 15:00 收件人 ***@***.***> 抄送至 ***@***.***>***@***.***> 主题 Re: [zotero-chinese/wiki] blog/posts/hello-zotero-7 (Discussion #261) 
占用的内存很大
这个是因为你的安装位置选的极为不合适，安装的时候自定义了路径，但没有给Zotero程序单独建立一个文件夹存放文件，导致Zotero程序文件所在目录的所有文件的大小都被统计在内了（比如装到了Program Files根目录，就会导致这个目录下所有软件都被认为是Zotero的一部分，又比如把数据文件夹放到了程序文件夹里，导致文献数据和PDF也被认为是程序文件的一部分）。这种情况最好卸载重装，为程序单独使用一个独立的目录存放。
为什么我的安装应用里有两个zotero
这个可能是Zotero本身升级的小问题，但如果你没有自定义安装路径的话是不会遇到的。简单来说就是升级的时候没能覆盖安装，而是另装了一个新的。综合前面占用的问题，建议不要自定义程序目录，毕竟你选的目录很可能非常不合适，不好说未来会不会遇到些抽象问题。
我可以删掉一个么
可以，有必要的话两个都卸了，重装个干净的在默认的路径下，别改。

**Reply by wakewon** [reply]
> 在D盘里专用建了一个zotero的文件夹，下载软件时就把自定义安装在了D盘的这个文件夹里，这样也不可以么？

可以，但不要把数据文件夹也选在这个路径下，而是应该另外新起一个独立的目录，单独用来存文献数据。详见 [中文社区文档：自定义 Zotero 数据存储位置](https://zotero-chinese.com/user-guide/faqs/custom-data-directory)

前面的回复也提到了，“又比如把数据文件夹放到了程序文件夹里，导致文献数据和PDF也被认为是程序文件的一部分”。

> 我担心卸载重装后我的文献还会存在么？

会存在，不影响。如果你卸载的时候不勾选 Remove my Zotero personal data and customizations 前的复选框，重装 Zotero 后数据和设置都不会丢失。考虑到你很可能把数据和程序混着放了，强烈建议你按照社区里的教程提前备份一份备用。

> 如果我使用默认安装位置的话，我觉得我的C盘会撑不住

其实不然，因为Zotero的程序和数据是分开放的，程序本身不是太大（虽然也不是很小），放在C盘问题不大。如果硬盘空间紧张，可以单独把数据存储位置放在其他盘，这样杂七杂八的问题能相对少一些（毕竟Zotero的抽象更新/安装机制在自定义了程序安装路径后时不时搞点大新闻，但只是改数据存储位置问题不大，注意事项也都在前面说的文档里写了）。当然，如果C盘空间极度紧张，非说装在别的盘行不行，那确实也行，但仍要做到程序文件和数据文件完全分离，放在两个不同的目录里。

**Reply by wakewon** [reply]
至于显示的那个巨大的占用空间，通常来说并不是一个可清理的缓存，而是把程序之外的其他文件也统计进去了。多出来的那些东西通常是不可删/不可清理的，Zotero也没什么可清理的缓存。那个数字并不重要，当然，不管怎样都强烈不建议混放程序和文献数据。
